src/parser/parser_visitor/__init__.py

```diff
@@ -0,0 +1,89 @@
+# coding=utf-8
+# Copyright 2018-2020 EVA
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from antlr4 import TerminalNode
+from src.parser.evaql.evaql_parserVisitor import evaql_parserVisitor
+from src.parser.evaql.evaql_parser import evaql_parser
+
+
+class ParserVisitor(evaql_parserVisitor):
+    from ._insert_statements import visitInsertStatement,\
+        visitUidList,\
+        visitInsertStatementValue
+
+    from ._create_statements import visitColumnCreateTable,\
+        visitCreateDefinitions,\
+        visitColumnDeclaration,\
+        visitColumnDefinition,\
+        visitUniqueKeyColumnConstraint,\
+        visitSimpleDataType
+
+    from ._create_statements import visitIntegerDataType,\
```

Comment:
Would this approach work if we import it as:
from ._create_statements import *

Reply:
It doesn't. There is an alternative using multiple inheritances. For instance, createStatement(evaql_parserVisitor)... and then ParserVisitor(createStatement, ..).
Should we move to this approach?